iron-collapse.html

```diff
@@ -239,6 +239,14 @@
       this.notifyResize();
     },
 
+    _isVisible: function() {
+      var rect = this.getBoundingClientRect();
```

Comment:
Yes, but the suggested version still checks the bounding rect, it just limit to check 1 dimension (width or height, see the getter of `dimension`), while yours checks for `left, right, top, bottom` too. This should not return false if top = 0